microsoft-edge/webview2/reference/dotnet/0-9-628/microsoft-web-webview2-core-corewebview2environment.md

```diff
@@ -23,16 +23,17 @@ This represents the WebView2 Environment.
 --------------------------------|---------------------------------------------
 [BrowserVersionString](#browserversionstring) | The browser version info of the current CoreWebView2Environment, including channel name if it is not the stable channel.
 [NewBrowserVersionAvailable](#newbrowserversionavailable) | NewBrowserVersionAvailable fires when a newer version of the Edge browser is installed and available for use via WebView2.
+[CompareBrowserVersions](#comparebrowserversions) | Compare browser versions correctly to determine which version is newer, older or same.
+[CreateAsync](#createasync) | Creates an evergreen WebView2 Environment using the installed Edge version.
 [CreateCoreWebView2CompositionControllerAsync](#createcorewebview2compositioncontrollerasync) | Asynchronously create a new WebView for use with visual hosting.
 [CreateCoreWebView2ControllerAsync](#createcorewebview2controllerasync) | Asynchronously create a new WebView.
 [CreateCoreWebView2PointerInfo](#createcorewebview2pointerinfo) | Create an empty CoreWebView2PointerInfo.
 [CreateWebResourceResponse](#createwebresourceresponse) | Create a new web resource response object.
+[GetAvailableBrowserVersionString](#getavailablebrowserversionstring) | Get the browser version info including channel name if it is not the stable channel or the Embedded Edge.
```

Comment:
```suggestion
[GetAvailableBrowserVersionString](#getavailablebrowserversionstring) | Get the browser version information. You also get the channel name if the channel is not a stable channel, or you are using the embedded version of Microsoft Edge.
```

Reply:
Does "embedded" mean the mobile version of Microsoft Edge?

Reply:
The 'Embedded Edge' here should be 'WebView2 Runtime'. There is no channel name for WebView2 Runtime, either.

Reply:
[GetAvailableBrowserVersionString](#getavailablebrowserversionstring) | Get the browser version information. You also get the channel name if the channel is not a stable channel.

Does that work?

Reply:
Can we also mention that end developers won't see channel name if they are using WebView2 Runtime?

Reply:
GetAvailableBrowserVersionString | Get the browser version information. You also get the channel name if the channel is not a stable channel. If you use the WebView Runtime, no channel name returned.